package.json

```diff
@@ -79,7 +80,7 @@
     "pretest:ci": "npm run build:app",
     "test:ci": "mocha --forbid-only --recursive ./test --require ./test/mocha-root-hooks.mjs --reporter mocha-multi-reporters --reporter-options configFile=./test/config/reporter.json",
     "build": "npm run build:app && npm run build:sea",
-    "build:app": "esbuild --bundle ./src/user-entrypoint.mjs --platform=node --outfile=./dist/cli.cjs --format=cjs --inject:./sea/import-meta-url.js --define:import.meta.url=importMetaUrl --define:process.env.NODE_ENV=\\\"production\\\"",
+    "build:app": "esbuild --loader:.node=file --bundle ./src/user-entrypoint.mjs --platform=node --outfile=./dist/cli.cjs --format=cjs --inject:./sea/import-meta-url.js --define:import.meta.url=importMetaUrl --define:process.env.NODE_ENV=\\\"production\\\"",
```

Comment:
tl;dr: What does this do?

Reply:
Some node modules like low level crypto libraries esbuild does not bundle by default. This instructs esbuild to do so by copying the files it needs into the bundle.

Reply:
There are specific versions published: https://hub.docker.com/r/fauna/faunadb/tags

So i could see a case for having the tag be mutable for debugging. So you'd take a `--tag` argument instead. But we could also just drop it and always do latest which seems more appropriate for a web service.